### ISSUE TITLE
RTRTR Duplicating annouces (rtrtr 0.2.1)

### ISSUE THREAD

**Issue by dagmoller** [NLnetLabs/rtrtr#72]
Output from routinator:
```
# curl -sk http://localhost:8323/json | grep 23.249.169.0
    { "asn": "AS203020", "prefix": "23.249.169.0/24", "maxLength": 32, "ta": "arin" },
```
    
Duplicated output from rtrtr:
```
# curl -sk http://localhost:8080/json | grep 23.249.169.0
    { "asn": "ASAS203020", "prefix": "23.249.169.0/24/24", "maxLength": 32, "ta": "N/A" },
    { "asn": "ASAS203020", "prefix": "23.249.169.0/24/24", "maxLength": 32, "ta": "N/A" },
```

This is causing erros on routers:
`18171321: May 25 08:34:21.233: %BGP-4-BGP_BAD_RPKI_MESSAGE: BGP received an invalid RPKI message from rpki server XXX.XXX.XXX.XXX/9001 Duplicate Announce Recieved for the Same Record Duplicate announce for prefix 23.249.169.0/24 ASN 203020 epoch 1739 hex dump: 00 04 00 00 00 00 00 14 01 18 20 00 17 F9 A9 00 00 03 19 0C`

**Comment by partim** [comment]
Thank you for the report!

Can you share which units and targets are involved in creating that output? Just the sequence of connected unit would be enough for now.

**Comment by rob-robertson** [comment]
i'm running into this bug with our Nokias.   Just running a cache with no clients, but referencing a routinator and a fort-validator, i'm seeing on start up, there are no dups.  the dups seem to correlate to updates.  the more updates, the more duplicate routes.

http-listen = ["0.0.0.0:8080"]

[units.routinator-3323]
type = "rtr"
remote = "64.125.64.4:3323"

[units.fort-3323]
type = "rtr"
remote = "64.125.64.3:3323"

[units.any-rtr]
type = "any"
random = false
sources = [ "routinator-3323" , "fort-3323" ]

[targets.local-3323]
type = "rtr"
listen = [ "0.0.0.0:3323" ]
unit = "any-rtr"


[targets.http-json]
type = "http"
path = "/json"
format = "json"
unit = "any-rtr"

**Comment by partim** [comment]
We have just published a release candidate for 0.2.2 that should fix this issue, if you want to give that a try. The final release will likely happen in the second half of next week.
